packages/nimble-components/src/combobox/tests/combobox.spec.ts

```diff
@@ -531,6 +552,30 @@ describe('Combobox', () => {
         await disconnect();
     });
 
+    it('when typing in value with inline autocomplete, option at bottom of list scrolls into view', async () => {
+        const { element, connect, disconnect } = await setupWithManyOptions(
+            undefined,
+            undefined,
+            ComboboxAutocomplete.inline
+        );
+        await connect();
+        await waitForUpdatesAsync();
+
+        const lastOption = element.options[element.options.length - 1]!;
+        await clickAndWaitForOpen(element);
+        updateComboboxWithText(element, '1');
+        await waitForUpdatesAsync();
+        // This second call seems necessary to allow the requestAnimationFrame in the FAST Combobox implementation
+        // time to run for the first update.
+        await waitForUpdatesAsync();
```

Comment:
I wasn't able to come up with a testing solution that didn't involve calling `waitForUpdatesAsync` twice in a row after the first update.